integration/integrate-using-generic-webhooks.adoc

```diff
@@ -0,0 +1,42 @@
+[id="integrate-using-generic-webhooks"]
+= Integrating by using generic webhooks
+include::modules/common-attributes.adoc[]
+:context: integrate-using-generic-webhooks
+
+toc::[]
+
+[role="_abstract"]
+With {product-title}, you can send alert notifications as JSON messages to any webhook receiver.
+When a violation occurs, {product-title} makes an HTTP POST request on the configured URL.
+The POST request body includes JSON-formatted information about the alert.
+
+The webhook POST request's JSON data includes a `v1.Alert` object and any custom fields that you configure.
+
+[source,json]
+----
+{
+  "alert": {
+    "id": "<id>",
+    "time": "<timestamp>",
+    "policy": {
+      "name": "<name>",
+      ...
+    },
+    ...
+  },
+  "<custom_field_1>": "<custom_value_1>"
+}
+----
+
+You can create multiple webhooks, for example, you can create a webhook for receiving all audit logs and a different webhook for alert notifications.
+
+To forward alerts from {product-title} to any webhook receiver:
+
+. Set up a webhook URL to receive alerts.
+. Use the webhook URL to set up notifications in {product-title}.
+. Identify policies for which you want to send notifications, and update the notification settings for those policies.
+
+include::modules/webhook-configuring-acs.adoc[leveloffset=+1]
+
+:toolname: webhook
+include::modules/configure-policy-notifications.adoc[leveloffset=+1]
```

Comment:
I'm seeing an error in the preview build for this module include. Perhaps the file hasn't been added? Here's the message:
> Unresolved directive in integrate-using-generic-webhooks.adoc - include::modules/configure-policy-notifications.adoc[leveloffset=+1]

Reply:
Its a common module that is part of another PR. https://github.com/openshift/openshift-docs/pull/34022/files#diff-677c9328c822a31b76b5600a631c05dff7cda6dc406910a889acff1e51c61898

Reply:
Gotcha, thanks.

Reply:
I think we need to merge https://github.com/openshift/openshift-docs/pull/34022 for this to resolve before we actually merge this PR, if that makes sense. Just to make sure build checks pass.